aten/src/ATen/native/mps/MPSGraphVenturaOps.h

```diff
@@ -3,6 +3,8 @@
 
 // TODO: Remove me when moved to MacOS 13
 @interface MPSGraph (VenturaOps)
+
+#if !defined(__MAC_10_13)
```

Comment:
Denis, is this correct? Shouldn't it be __MAC_13_0... something? Also can you make this check :
```
#if defined () && < Mac_version < 130000

#endif
```